frontend/src/views/admin/DashboardPage/DashboardPage.tsx

```diff
@@ -28,37 +49,106 @@ export const AdminDashboardPage = () => {
     }
   }, [isNotAllowed, isUserLoading]);
 
+  useEffect(() => {
```

Comment:
You can just remove this `useEffect`. Set allowspecificDomain is useless state as the config already contains the state.

Just use tertiary operator inside select below

Reply:
eslint is not liking that 
<img width="743" alt="Screenshot 2024-02-12 at 4 04 47 AM" src="https://github.com/Infisical/infisical/assets/40920317/6761ee2f-f121-4502-9344-1dceda091979">

Reply:
Nester ternary was little clean IMO
```
  const signUpStatus = config.allowSignUp
  ? config.inviteOnlySignUp && "invite-only"
  : "disabled";

  const signUpType = signUpStatus || "anyone";
  
  ```